test/fixtures/label/borderDash.js

```diff
@@ -0,0 +1,64 @@
+module.exports = {
+  tolerance: 0.0055,
+  config: {
+    type: 'bar',
+    options: {
+      scales: {
+        x: {
+          labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+        },
+        y: {
+          display: false,
+          min: 0,
+          max: 25
+        }
+      },
+      plugins: {
+        annotation: {
+          annotations: {
+            text1: {
+              type: 'label',
+              xScaleID: 'x',
+              yScaleID: 'y',
+              xValue: 3.5,
+              yValue: 10,
+              backgroundColor: 'rgba(33, 101, 171, 0.5)',
+              borderColor: 'rgb(33, 101, 171)',
+              borderDash: [6, 6],
+              borderWidth: 1,
+              content: 'This is my text'
+            },
+            text2: {
+              type: 'label',
+              xScaleID: 'x',
+              yScaleID: 'y',
+              xValue: 'February',
+              yValue: 11,
+              backgroundColor: 'rgba(101, 33, 171, 0.5)',
+              borderColor: 'rgb(101, 33, 171)',
+              borderDash: [6, 6],
+              borderWidth: 1,
```

Comment:
Should make the border wider for this test, so any regression is more likely catched